src/third-parties/google-analytics/data.json

```diff
@@ -5,15 +5,15 @@
   "scripts": [
     {
       "url": "https://www.googletagmanager.com/gtag/js",
-      "params": ["id"],
+      "params": ["id", "l"],
```

Comment:
passing in the "l" param for the custom datalayername (thanks @huang-julien)

Reply:
Just to confirm, `l` is a URL query param correct? I think the `params` property here assigns URL params if I'm not mistaken

Reply:
Yup, see https://developers.google.com/tag-platform/tag-manager/datalayer#rename_the_data_layer